src/Microsoft.HttpRepl.Tests/Commands/GetCommandTests.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.Tests.SampleApi;
+using Microsoft.Repl;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.Commands
+{
+    public class GetCommandTests : IClassFixture<GetCommandsFixture>
+    {
+        private readonly GetCommandsConfig _config;
+        public GetCommandTests(GetCommandsFixture getCommandsFixture)
+        {
+            _config = getCommandsFixture.Config;
+        }
+
+        [Fact]
+        public async Task NoBasePath_ReturnsMessage()
+        {
+            HttpState httpState = new HttpState();
+
+            string expectedErrorMessage = StringResources.Error_NoBasePath.SetColor(httpState.ErrorColor);
+            string actualErrorMessage = null;
+
+            IShellState shellState = MockHelpers.GetMockedShellState(errorMessageCallback: (s) => actualErrorMessage = s);
+
+            GetCommand command = new GetCommand();
+            ICoreParseResult parseResult = CoreParseResultHelper.Create("GET");
+            CancellationTokenSource cts = new CancellationTokenSource();
+
+            await command.ExecuteAsync(shellState, httpState, parseResult, cts.Token);
+
+            Assert.Equal(expectedErrorMessage, actualErrorMessage);
+        }
+
+        [Fact]
+        public async Task MultipartRoute()
+        {
+            int expectedResponseLines = 5; // The path, three headers and the response.
+            string expectedResponseContent = "This is a test response.";
+            List<string> actual = new List<string>();
+            
+            IShellState shellState = MockHelpers.GetMockedShellState(writeLineCallback: (s) => actual.Add(s));
+
+            HttpState httpState = HttpStateHelpers.Create(_config.BaseAddress, "this/is/a/test/route");
+
+            GetCommand command = new GetCommand();
+            ICoreParseResult parseResult = CoreParseResultHelper.Create("GET");
+            CancellationTokenSource cts = new CancellationTokenSource();
+
+            await command.ExecuteAsync(shellState, httpState, parseResult, cts.Token);
+
+            Assert.Equal(expectedResponseLines, actual.Count); 
+            Assert.Equal(expectedResponseContent, actual[expectedResponseLines - 1]);
+
```

Comment:
nit: Empty line after Assert here and next method

Reply:
Removed in e3302fb